client/src/components/App.jsx

```
@@ -1,15 +1,18 @@
+import Homepage from "./Homepage";
+import { useEffect } from "react";
 import { useEffect, useMemo, Suspense } from "react";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'useEffect'. Consider to delete it or rename it.

'useEffect' is defined here:


(lint/suspicious/noRedeclare)